src/components/HypothesisBuilder/comments.js

```diff
@@ -69,69 +32,39 @@ export function CommentsSection({
 
   useEffect(() => {
     getPersons().then(({ data }) => {
-      const usersDate = data.map((user) => ({ ...user, display: user.name }));
+      const usersDate = data.map((user) => ({
+        ...user,
+        display: user.name,
+        icon: user.avatar
+      }));
       setUsers(usersDate);
     });
   }, []);
 
-  const renderSuggestion = useCallback(
-    ({ name, avatar }) => (
-      <div className="flex items-center">
-        <Icon name={avatar} size="xl" />
-        <p className="pl-2">{name}</p>
-      </div>
-    ),
-    []
-  );
-
-  const onClickUserTag = (userId) => {
-    console.log("userId", userId);
+  const onClickUserTag = (type, id) => {
+    console.log("type tag", type, "value", id);
   };
 
-  const swapTags = (text) => {
-    const tags = text.match(/@\[.*?\]\(user:.*?\)/gi) || [];
-    const otherText = text.split(/@\[.*?\]\(user:.*?\)/gi);
-    return tags.reduce(
-      (display, myTag, index) => {
-        const tagDisplay = myTag.match(/\[.*?\]/gi)[0].slice(1, -1);
-        const tagData = myTag.match(/\(user:.*?\)/gi)[0].slice(6, -1);
-        return [
-          ...display,
-          <button
-            type="button"
-            key={tagData}
-            onClick={() => onClickUserTag(tagData)}
-            className="bg-blue-200 rounded"
-          >
-            {tagDisplay}
-          </button>,
-          otherText[index + 1]
-        ];
-      },
-      [otherText[0]]
-    );
-  };
+  const commentData = useMemo(
+    () => [
+      {
+        data: users,
+        markup: "@[__display__](user:__id__)",
+        trigger: "@"
```

Comment:
Can we move markup and trigger as attributes on Comment Input and then default them ?

Reply:
We can if we use only mentions of users. But if you need to make tags mentions and user mentions then the current example will be better

Reply:
Even if we add support for tags, it would be in addition `#` to users not instead of

Reply:
Do you suggest doing the following?
```
<CommentInput data={data} markup="@[__display__](user:__id__)" trigger="@" ... />
```
But how then to make support a few triggers and markup?
In this case, we can use either @ or # trigger. But not both at the same time

Reply:
I suggest that interface but default the markup and trigger, at a later point in the future if we decide to handle tags we can come up with a solution then

Reply:
Ok I updated interface